profiler/src/ProfilerEngine/Datadog.Profiler.Native/PInvoke.cpp

```diff
@@ -45,11 +45,6 @@ extern "C" void __stdcall ThreadsCpuManager_Map(std::uint32_t threadId, const WC
     profiler->GetThreadsCpuManager()->Map(threadId, pName);
 }
 
-extern "C" BOOL __stdcall GetAssemblyAndSymbolsBytes(void** ppAssemblyArray, int* pAssemblySize, void** ppSymbolsArray, int* pSymbolsSize, WCHAR* moduleName)
```

Comment:
GetAssemblyAndSymbolsBytes should be removed from profiler\src\ProfilerEngine\Datadog.Profiler.Native.Windows\Datadog.Profiler.Native.def